RadixWallet/Features/AssetsFeature/Components/PoolUnitsList/Components/LSUResource/LSUStake.swift

```diff
@@ -97,6 +107,12 @@ public struct LSUStake: FeatureReducer {
 			}
 
 			// TODO: Show details
```

Comment:
is this todo comment still relevant?

Reply:
definitely not.

Reply:
That was probably my comment when I realised we weren't showing the details yet